app/components/works/keywords_row_component.html.erb

```diff
@@ -3,7 +3,10 @@
     <%= form.label :label, 'Keyword', class: 'col-form-label sr-only' %>
     <div data-controller="autocomplete autocomplete-edit" data-autocomplete-url-value="/autocomplete" data-autocomplete-min-length-value="3" class="dropdown">
       <%= form.text_field :label, { class: "form-control#{ 'is-invalid' if error?}", required: true, 'data-autocomplete-target': 'input', 'data-autocomplete-edit-target': 'input' } %>
-      <%= form.hidden_field :uri, { required: true, 'data-autocomplete-target': 'hidden', 'data-autocomplete-edit-target': 'uri', 'data-action': 'autocomplete-edit#change' }%>
+      <%#= form.hidden_field :value, { 'data-autocomplete-target': 'hidden', 'data-autocomplete-edit-target': 'value', 'data-action': 'autocomplete-edit#change' }%>
+      <input type="hidden" data-autocomplete-target="hidden" data-autocomplete-edit-target="value" data-action="autocomplete-edit#change" />
```

Comment:
What is the purpose of this? How can a hidden field trigger a change?

Reply:
This hidden input is populated by the autocomplete controller (recall, an instance of stimulus-autocomplete). In turn this triggers the autocomplete-edit controller to split the value and populate the uri and cocina type inputs.

The autocomplete controller changing the hidden field does trigger the change -- not sure how.

Reply:
Can you add a comment about this mechanism?

Reply:
Done.